api/server/router/swarm/cluster_routes.go

```diff
@@ -132,7 +132,7 @@ func (sr *swarmRouter) createService(ctx context.Context, w http.ResponseWriter,
 
 	id, err := sr.backend.CreateService(service, encodedAuth)
 	if err != nil {
-		logrus.Errorf("Error creating service %s: %v", id, err)
+		logrus.Errorf("Error creating service: %v", err)
```

Comment:
If there's less chance that the name is empty, I think adding it would be good. Anything is better than "nothing"?